script/install.sh

```diff
@@ -9,14 +9,37 @@ elif [ "${BOXEN_HOME:-}" != "" ] ; then
   prefix=${BOXEN_HOME:-}
 fi
 
-mkdir -p $prefix/bin
-rm -rf $prefix/bin/git-lfs*
+local_install=false
+
+while [[ $# -gt 0 ]]; do
+  case "$1" in
+    --local)
+      local_install=true
+      prefix="$HOME/.local"
+      shift
+      ;;
+    *)
+      echo "Unknown option: $1"
+      exit 1
+      ;;
+  esac
+done
+
+# Check if the user has permission to install in the specified prefix
+if [ ! -w "$prefix" ]; then
+  echo "Error: Insufficient permissions to install in $prefix. Try running with sudo or choose a different prefix."
```

Comment:
Let's redirect this to standard error with `>&2` so that errors go the right place.